_pages/leaderboard.md

```diff
@@ -5,10 +5,10 @@ permalink: /leaderboard
 
 | Name          | Iced          | Got Iced   |
 | ------------- |:-------------:|:----------:|
-| Andy          | 12            | 2          |
+| Andy          | 12            | 3          |
 | Sam           | 3             | 2          |
 | Jacky         | 2             | 7          |
-| Ben           | 1             | 2          |
+| Ben           | 2             | 2          |
```

Comment:
@BenEmdon wait I think you deserve to get +2 here because he had to drink both

Reply:
I don't think andy actually drank the litre

Reply:
Oh wait nvm. Andy asked Ben to be blocked with an regular ice instead, that liter was from Bernard. All good here.

Reply:
Did he drink both tho?